src/components/Tool/Appeal/Flow/ContactFlowColumn/ContactFlowColumn.tsx

```diff
@@ -56,27 +57,52 @@ export const ContactFlowColumn: React.FC<Props> = ({
   onContactSelected,
   changeContactStatus,
 }) => {
-  const { appealId, sanitizedFilters } = React.useContext(
-    AppealsContext,
-  ) as AppealsType;
+  const {
+    appealId,
+    sanitizedFilters,
+    starredFilter,
+    selectMultipleIds,
+    deselectMultipleIds,
+  } = React.useContext(AppealsContext) as AppealsType;
   const { t } = useTranslation();
 
   const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
   const open = Boolean(anchorEl);
 
+  const contactsFilters = useMemo(
+    () => ({
+      ...sanitizedFilters,
+      ...starredFilter,
+      appeal: [appealId ?? ''],
+      appealStatus,
+      wildcardSearch: searchTerm as string,
+    }),
+    [sanitizedFilters, starredFilter, searchTerm, appealId],
```

Comment:
Should we add `appealStatus` to the dependency array? I guess it's not likely to change at runtime.

Reply:
The appealStatus won't change